lisa/tools/ip.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from __future__ import annotations
+
+from assertpy import assert_that
+
+from lisa.executable import Tool
+from lisa.tools import Cat
+
+
+class Ip(Tool):
+    @property
+    def command(self) -> str:
+        return "ip"
+
+    def _check_exists(self) -> bool:
+        return True
+
+    def restart_device(self, nic_name: str) -> None:
+        self.node.execute(
+            f"ip link set dev {nic_name} down;ip link set dev {nic_name} up", shell=True
```

Comment:
Can we use `self.run` here instead of `self.node.execute`?

Reply:
It has two commands, and joined with the `;`. The `run` makes it looks weird.